usefulUtilities/hiFiCalendar/README.md

```diff
@@ -73,6 +73,10 @@ Define your permitted access locations to include http://localhost under Domain
 
 # Release Notes
 
+## 3.5 | [commit 43d222f] | 2019-06-26
```

Comment:
does this work without the link inside () like the others?

Reply:
Oh, to be honest, I kind of gave up on that.  In fact, I think all the links on the readme are wrong.  They all reference pull 420...hehe